### ISSUE TITLE
Search Button and Functionality

### ISSUE THREAD

**Issue by MJC598** [MJC598/RJI-Software-Engineering-Project#9]
Description: Search bar on the main page of the website that has no present functionality. Need to fix this so that we can submit the HTML form to search the server for the requested query information.

To Reproduce:
1. Visit the ec2 public DNS page at http://ec2-13-59-146-242.us-east-2.compute.amazonaws.com/
2. Type query information in the search bar in plain english
3. Press the enter key

Expected Behavior: The expected behavior of the search bar is to type in the information you would like to query the database for, hit enter, and then have the information you wanted, or any related information, displayed to the user.

Desktop:
OS: All OSes
Browser: All browsers

**Comment by Deimious** [comment]
We've been unable to work on this portion of the project for about a week now because we haven't had a library of classified images to search from. Once we get the classifier situated and functioning properly, we should be able to fix the search functionality easily.
